config/locales/en.yml

```diff
@@ -1375,6 +1423,14 @@ en:
       policies:
         update: "Update Policy Chain"
     labels:
+      account:
+        payment_gateway_type: Gateway
+
+      numbering_period: Billing periods for invoice IDs
+      finance_billing_strategy:
```

Comment:
```suggestion
      finance/billing_strategy:
```

Reply:
Ooops, actually it breaks it!

I have no idea why `hints` and `labels` do not behave the same way :smiling_face_with_tear:

Reply:
Wow nice finding. I thought it was always made by underscores.